lua/tardis/themes/visgui/default-transparent.lua

```diff
@@ -29,6 +29,10 @@ local theme = {
 		scanner = "screen_scanner.png",
 		settings = "screen_settings.png",
 		["TARDIS Status"] = "screen_status.png",
+
```

Comment:
Probably shouldn't be a new line here breaking the two sections up, same below as well

Reply:
Fixed